build/teamcity/cockroach/ci/tests/acceptance_impl.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -xeuo pipefail
+
+bazel build //pkg/cmd/bazci //pkg/acceptance:acceptance_test //pkg/cmd/cockroach-short:cockroach-short --config=ci
+find $(bazel info bazel-bin)/.. -ls
+cp \
+    $(bazel info bazel-bin)/pkg/acceptance/acceptance_test_/acceptance_test \
+    $(bazel info bazel-bin)/pkg/cmd/cockroach-short/cockroach-short_/cockroach-short \
```

Comment:
A couple nits:

1. When doing `bazel info bazel-bin`, add `--config=crosslinux`
2. Only do `bazel info bazel-bin --config=crosslinux` once, save it in a variable, and reference that instead of calling into it multiple times. `bazel info` can take a second or so.